bee-message/src/payload/fpc/mod.rs

```diff
@@ -3,25 +3,54 @@
 
 //! Module describing the FPC statement payload.
 
-mod conflicts;
-mod timestamps;
+mod conflict;
+mod timestamp;
 
-pub use conflicts::{Conflict, Conflicts};
-pub use timestamps::{Timestamp, Timestamps};
+pub use conflict::Conflict;
+pub use timestamp::Timestamp;
 
-use crate::{payload::MessagePayload, MessageUnpackError, ValidationError};
+use crate::{
+    payload::{transaction::TransactionId, MessagePayload, PAYLOAD_LENGTH_MAX},
+    MessageId, MessageUnpackError, ValidationError,
+};
+use bee_packable::{error::UnpackPrefixError, BoundedU32, Packable, VecPrefix};
 
-use bee_packable::Packable;
+use core::convert::Infallible;
+
+/// No [`Vec`] max length specified, so use [`PAYLOAD_LENGTH_MAX`] / length of [`Conflict`].
+const PREFIXED_CONFLICTS_LENGTH_MAX: u32 =
+    PAYLOAD_LENGTH_MAX / (TransactionId::LENGTH + 2 * core::mem::size_of::<u8>()) as u32;
+
+/// No [`Vec`] max length specified, so use [`PAYLOAD_LENGTH_MAX`] / length of
+/// [`Conflict`](crate::payload::fpc::Conflict).
+const PREFIXED_TIMESTAMPS_LENGTH_MAX: u32 =
+    PAYLOAD_LENGTH_MAX / (MessageId::LENGTH + 2 * core::mem::size_of::<u8>()) as u32;
+
+fn unpack_prefix_to_timestamp_validation_error(error: UnpackPrefixError<Infallible>) -> ValidationError {
```

Comment:
I am being extremely picky now but can you put this function after the conflict one please ?